src/components/GeriLifeTopbar.vue

```diff
@@ -72,43 +41,59 @@
           :options="languageOptions"
         />
       </div>
-      <div class="col-9 col-md-4" v-if="!!getCookie('token')">
-        <q-btn
-          @click="logoutAndRedirect"
-          :size="isSmOrAbove ? 'md' : 'xs'"
-          outline
-          class="float-right flat text-black bg-white"
-        >
-          {{ $i18n.t("logout-logoutButton") }}
-        </q-btn>
-
-        <q-btn
-          @click="$router.push({ path: '/settings' })"
-          class="float-right text-grey-7"
-          flat
-          :size="isSmOrAbove ? 'md' : 'xs'"
-          icon="fa fa-cog"
-        >
-          <span class="q-ml-sm">{{
-            $i18n.t("mainLayoutNavbar-settingsLink")
-          }}</span>
-        </q-btn>
 
-        <q-btn
-          @click="openProfile = true"
-          class="float-right text-grey-7"
-          flat
-          :size="isSmOrAbove ? 'md' : 'xs'"
-          icon="fa fa-user"
-        >
-          <span class="q-ml-sm">{{
-            $i18n.t("mainLayoutNavbar-profileLink")
-          }}</span>
-        </q-btn>
-      </div>
-    </div>
-    <change-password v-if="openProfile" @close="openProfile = false" />
-  </q-toolbar>
+      <q-btn
+        v-if="!!getCookie('token')"
+        @click="logoutAndRedirect"
+        outline
+        class="q-ml-xs flat text-black bg-white"
+      >
+        {{ $i18n.t("logout-logoutButton") }}
+      </q-btn>
+      <q-btn
+        v-if="!!getCookie('token') && !isMdOrAbove"
+        flat
+        @click="drawerRight = !drawerRight"
+        round
+        dense
+        icon="menu"
+      />
+      <change-password v-if="openProfile" @close="openProfile = false" />
+    </q-toolbar>
+    <q-drawer
+      v-if="!!getCookie('token') && !isMdOrAbove"
+      side="right"
+      v-model="drawerRight"
+      :width="200"
+      bordered
+    >
+      <q-scroll-area class="fit">
+        <q-list>
+          <template v-for="(menuItem, index) in menu">
+            <q-item
+              :class="{
+                'text-black': selected !== menuItem.title,
+                'text-white': selected === menuItem.title,
+              }"
+              :key="index"
+              clickable
+              @click="clicked(menuItem, index)"
+              active-class="bg-primary"
+              :active="menuItem.title === selected"
+            >
+              <q-item-section avatar>
+                <q-icon :name="menuItem.icon" />
+              </q-item-section>
+              <q-item-section>
+                {{ menuItem.title }}
+              </q-item-section>
+            </q-item>
+            <q-separator :key="'sep' + index" v-if="menuItem.separator" />
+          </template>
+        </q-list>
```

Comment:
Just add the profile link here as a separate entity.

Reply:
E.g. we can use the `q-dialogue` prompt:

https://quasar.dev/vue-components/dialog#usage

```html
    <q-dialog v-model="prompt" persistent>
      <q-card style="min-width: 350px">
        <q-card-section>
          <div class="text-h6">Your address</div>
        </q-card-section>

        <q-card-section class="q-pt-none">
          <q-input dense v-model="address" autofocus @keyup.enter="prompt = false" />
        </q-card-section>

        <q-card-actions align="right" class="text-primary">
          <q-btn flat label="Cancel" v-close-popup />
          <q-btn flat label="Add address" v-close-popup />
        </q-card-actions>
      </q-card>
    </q-dialog>
```

Reply:
it is already there